src/hotkeys/Localisation/zhCN.xml

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ArrayOfModLocEntry xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
+    <ModLocEntry key="btn.enable">启用</ModLocEntry>
+    <ModLocEntry key="ais.header">AI和隐匿</ModLocEntry>
+    <ModLocEntry key="ais.btn.enable.title">启用AI和隐匿热键</ModLocEntry>
+    <ModLocEntry key="ais.kb.ai.title">AI热键</ModLocEntry>
+    <ModLocEntry key="ais.kb.stealth.title">隐匿热键</ModLocEntry>
+
+    <ModLocEntry key="ws.btn.enable.title">启用武器栏热键</ModLocEntry>
+    <ModLocEntry key="ws.header">武器栏</ModLocEntry>
+    <ModLocEntry key="ws.kb.00.title">武器 1</ModLocEntry>
+    <ModLocEntry key="ws.kb.01.title">武器 2</ModLocEntry>
+    <ModLocEntry key="ws.kb.02.title">武器 3</ModLocEntry>
+    <ModLocEntry key="ws.kb.03.title">武器 4</ModLocEntry>
+    <ModLocEntry key="ws.kb.cycle.title">切换武器</ModLocEntry>
+    <ModLocEntry key="ws.toggle.for.all.title">对所有选定的角色</ModLocEntry>
+    <ModLocEntry key="ws.toggle.inventory.desc">如果启用，武器栏的功能将在库存界面上应用。</ModLocEntry>
+    <ModLocEntry key="ws.toggle.inventory.title">启用库存界面的武器栏热键</ModLocEntry>
+    <ModLocEntry key="ws.toggle.inventory.for.all.desc">如果启用，在库存界面时,武器栏功能将应用于打开界面前所有选定的角色。</ModLocEntry>

```

Comment:
Could you change that to "If enabled the weapon set functions will be applied to all selected characters when on the inventory screen". After going through my localisation again I realized some of them don't make sense.